test/unit/render-util.test.js

```diff
@@ -20,6 +21,46 @@ describe('core/render/utils', () => {
     });
   });
 
+  // getAndRemoveDocisfyIgnorConfig()
+  // ---------------------------------------------------------------------------
+  describe('getAndRemoveDocisfyIgnorConfig()', () => {
+    test('getAndRemoveDocisfyIgnorConfig from <!-- {docsify-ignore} -->', () => {
+      const { content, ignoreAllSubs, ignoreSubHeading } =
+        getAndRemoveDocisfyIgnorConfig(
+          'My Ignore Title<!-- {docsify-ignore} -->'
+        );
+      expect(content).toBe('My Ignore Title');
+      expect(ignoreSubHeading).toBeTruthy();
+      expect(!!ignoreAllSubs).toBeFalsy();
+    });
```

Comment:
Because the jest can not handle `undefined` directly, change to verify in this way.